internal/pkg/template/manifest/manifest.go

```diff
@@ -94,3 +99,7 @@ func (m *Manifest) NamingTemplate() naming.Template {
 func (m *Manifest) IsObjectIgnored(_ model.Object) bool {
 	return false
 }
+
+func (m *Manifest) MainConfig() *model.ConfigKey {
```

Comment:
Ne že by to mělo být špatně, jen se pro info zeptám. Když nebude vyplněný, tak to tady vrátí `nil`? Nejsem si teď úplně jistý co se stane když to bude v manifestu chybět.

Reply:
Jj, je volitelny, aj v API specifikacii... dik za kontrolu.

![image](https://user-images.githubusercontent.com/19371734/165915856-9dde68a9-15e9-4039-8e42-ebbfae4111b9.png)

Reply:
jo jo, vím. I vidím `omitempty` v definici, jen jsem si teď nebyl jistej jak to dopadne technicky. 🙂 👍